src/infrastructure/group-store/local-group-store.ts

```diff
@@ -1,32 +1,45 @@
-import { Group } from "../../topics/group";
+import { FetchedData, Group } from "../../topics/group";
 import GroupStore from "../../topics/group/group.store";
 import { LocalFileStore } from "../file-store";
 
 export class LocalGroupStore extends GroupStore {
   localFileStore: LocalFileStore;
 
-  constructor(localFileStore?: LocalFileStore) {
+  constructor(prefix?: string) {
     super();
-    this.localFileStore = localFileStore
-      ? localFileStore
-      : new LocalFileStore("groups");
+    prefix = prefix ? prefix : "groups";
```

Comment:
Would be better to use the Nullish coalescing operator: `prefix ?? "groups"`

Reply:
changed ! thanks